dash/development/_py_components_generation.py

```diff
@@ -87,18 +84,40 @@ def __init__(self, {default_argtext}):
         description=description,
         prop_reorder_exceptions=prop_reorder_exceptions,
     ).replace("\r\n", "\n")
+    required_args = required_props(filtered_props)
+    is_children_required = 'children' in required_args
+    required_args = list(filter(lambda arg: arg != 'children', required_args))
```

Comment:
```suggestion
    required_args = [arg for arg in required_args if arg != "children"]
```
Not a big deal, but list comprehensions are a bit clearer (and shorter) than lambdas.

Reply:
Thanks! It looks better 😁 Fixed at
 https://github.com/plotly/dash/pull/2218/commits/13ac4c69b273ee5fe74c4421d4124f62ffec172a